README.md

```diff
@@ -111,9 +111,9 @@ there is no carry-over and no automatic ticket assignment by auto-review.
 
 ### Combine auto-review and openqa-investigate
 
-A possible approach to combine handling known issues and unknown issues is to
+A possible approach to combine handling known issues and unreviewed issues is to
```

Comment:
this is where it reads less nice than in before. In before we had "known and unknown" issues, not it's "known and unreviewed". Makes me think, should we use just "unknown" issues consistently everywhere and leave that aspect of "review" which is more like a SUSE internal convention what it means out of here?

Reply:
SUSE internal? To my mind unreviewed just means nobody has looked at the issue. I don't see a difference between unknown and unreviewed, though, so like I said, feel free to convince others and I will change everything to use "unknown".